pkg/vm/memory/memory_value.go

```diff
@@ -194,19 +183,65 @@ func (mv *MemoryValue) Equal(other *MemoryValue) bool {
 	return false
 }
 
-// Adds two memory values is the second one is a Felt
+// // Adds two memory values is the second one is a Felt
+// func (mv *MemoryValue) Add(lhs, rhs *MemoryValue) (*MemoryValue, error) {
+// 	var err error
+// 	if lhs.IsAddress() {
+// 		if !rhs.IsFelt() {
+// 			return nil, errors.New("rhs is not a felt")
+// 		}
+// 		// lhs : Address, rhs : Address
+// 		mv.address, err = mv.address.Add(lhs.address, rhs.felt)
+// 	} else {
+// 		if rhs.IsAddress() {
+// 			mv.address, err = mv.address.Add(rhs.address, lhs.felt)
+// 		} else {
+// 			mv.felt = mv.felt.Add(lhs.felt, rhs.felt)
+// 		}
+// 	}
+
+// 	if err != nil {
+// 		return nil, err
+// 	}
+// 	return mv, nil
+// }
```

Comment:
Why are we leaving this comment here?

Reply:
I will erase. I tried to compare it side by side before / after the change